### PR TITLE
Update `cpvpa` to `k8s.gcr.io/cpa/cpvpa:v0.8.4`

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area networking
/kind enhancement

**What this PR does / why we need it**:
Update `cpvpa` to `k8s.gcr.io/cpa/cpvpa:v0.8.4` since it's a multi-arch image.

**Special notes for your reviewer**:
/invite @acumino 

**Release note**:

```other operator
Update `cpvpa` to `k8s.gcr.io/cpa/cpvpa:v0.8.4` since it's a multi-arch image.
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -36,4 +36,4 @@ images:
 - name: calico-cpva
   sourceRepository: github.com/kubernetes-sigs/cluster-proportional-vertical-autoscaler
   repository: k8s.gcr.io/cpvpa-amd64
-  tag: v0.8.3
+  tag: v0.8.4
```
Comment (ialidzhikov) [inline charts/images.yaml:39]: I believe we have to change the repository to be k8s.gcr.io/cpa/cpvpa. The current repository k8s.gcr.io/cpa/cpvpa-amd64 does not seem to be multi-arch.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Review by acumino — APPROVED** [review]
/lgtm

**Review by ialidzhikov — COMMENTED** [review]
The corresponding image cannot be pulled 👀 

```
$ docker pull k8s.gcr.io/cpvpa:v0.8.4
Error response from daemon: manifest for k8s.gcr.io/cpvpa:v0.8.4 not found: manifest unknown: Failed to fetch "v0.8.4" from request "/v2/cpvpa/manifests/v0.8.4".
```

File: charts/images.yaml [hunk 2]
```diff
@@ -35,5 +35,5 @@ images:
   tag: 1.8.3
 - name: calico-cpva
   sourceRepository: github.com/kubernetes-sigs/cluster-proportional-vertical-autoscaler
-  repository: k8s.gcr.io/cpvpa-amd64
-  tag: v0.8.3
+  repository: k8s.gcr.io/cpvpa
```
Comment (ialidzhikov) [inline charts/images.yaml:38]:
```suggestion
  repository: k8s.gcr.io/cpa/cpvpa
```

Comment (acumino) [inline charts/images.yaml:38]:
```suggestion
  repository: k8s.gcr.io/cpa/cpvpa

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Comment by rfranzke** [comment]
Thanks!

**Pull request merged**